services/appnavstate/impl/src/main/kotlin/io/element/android/services/appnavstate/impl/DefaultAppNavigationStateService.kt

```diff
@@ -167,6 +166,10 @@ class DefaultAppNavigationStateService @Inject constructor(
         state.getAndUpdate { it.copy(navigationState = NavigationState.Root) }
     }
 
+    private fun logError(logPrefix: String) {
+        Timber.tag(loggerTag.value).w("$logPrefix must be call first, can't leave.")
```

Comment:
I think the message is not generic enough.

```suggestion
        Timber.tag(loggerTag.value).w("$logPrefix must be call first.")
```